core/src/main/scala/org/apache/spark/internal/config/Connect.scala

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.internal.config
+
+private[spark] object Connect {
```

Comment:
As Spark connect is build as plugin, all of the configuration should ideally be not located in core. Is there a way we can move this to the connect module?

Reply:
I agree. let's see other reviewer's suggestions on this.

Reply:
Yeah, it should better be placed within `connect` if possible. How do we use this plugin? I suspect the jar should be provided anyway (?)

Reply:
Does Spark Connect module depend on Spark Core? If yes, we can move this object to Spark Connect module.

Reply:
Yes it depends on core. Let me move this config and also address other comments.

Reply:
> Yeah, it should better be placed within `connect` if possible. How do we use this plugin? I suspect the jar should be provided anyway (?)

I believe there is a way to decide if the plugin is loaded and there is a default way of if this is loaded or not. Let me check that.

Reply:
Moved to connect module.